src/test/java/com/flightstats/hub/dao/TtlEnforcerTest.java

```diff
@@ -0,0 +1,103 @@
+package com.flightstats.hub.dao;
+
+import com.flightstats.hub.model.ChannelConfig;
+import com.flightstats.hub.util.Commander;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.mockito.Mock;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import static org.mockito.Mockito.mock;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+@Slf4j
+class TtlEnforcerTest {
+    // verify each time that the enforcer ignores this file
+    private static final String LOST_AND_FOUND_DIR = "lost+found";
+
+    @Mock
+    private ChannelService channelService;
+    @Mock
+    private Commander commander;
+
+    private TtlEnforcer ttlEnforcer;
+
+    private final Consumer<ChannelConfig> callback = config -> {
+    };
+
+    private String[] getSpokePathList(File spoke) {
+        return Optional.ofNullable(spoke.list()).orElse(new String[] {});
+    }
+
+    private boolean createMockSpokeFiles(File dir, String... fileNames) {
+        return Stream.of(fileNames).allMatch(str -> {
+            try {
+                return new File(dir.toPath() + "/" + str).createNewFile();
+            } catch (Exception e) {
+                log.error("error adding files to spoke path {}", e);
+                return false;
+            }
+        });
+    }
+
+    @BeforeEach
+    void setup() {
+        initMocks(this);
+        ttlEnforcer = new TtlEnforcer(commander);
+    }
+
+    @Test
+    void enforce_deletesFiles_affirmThreeFilesDeleted(@TempDir File spoke) {
```

Comment:
I might rename this to make it explicit that it's deleting directories that don't match up with channels or LOST_AND_FOUND_DIR rather than just deleting files.